.github/workflows/black.yml

```diff
@@ -0,0 +1,20 @@
+name: black
+
+on:
+  push:
+    paths:
+      - '**.py'
+  pull_request:
+    paths:
+      - '**.py'
+
+jobs:
+  python-black:
+    name: Python Black
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Python Black
+        uses: cytopia/docker-black@0.8
+        with:
+          path: '.'
```

Comment:
This misses the python files in the `bin/` directory.
I've added a fixup commit that adds entries for the python files in that directory.
If you agree with that change it would be great if you would squash into your commit.